Scenarios/Informative_Image_Selection_FR_Pattern/.gitignore

```diff
@@ -0,0 +1,119 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Can you add a requirements.txt file for this project and keep it in the root of this solution folder

Reply:
Can you also please add a root README entry to this solution in the Example Scenarios section

Reply:
Also add yourself to the Contributors section in the root README